### ISSUE TITLE
DotNet Run on a new F# Project produces no output

### ISSUE THREAD

**Issue by gavdraper** [dotnet/cli#1279]
I've just installed the latest DotNet CLI on my Macbook this is the output from DotNet --Version

 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 Runtime Id:  osx.10.11-x64

I've created a new F# application but am not seeing any output these are the steps I took to produce the problem.

DotNet New -l F#
DotNet Restore
DotNet Run

I get no errors with any of the above steps but I see no output from DotNet Run, I am expecting to see the output of the printfn in Program.fs.

See attached image for a better example

![screen shot 2016-02-07 at 19 22 42](https://cloud.githubusercontent.com/assets/971016/12874680/379ff8e6-cdd0-11e5-8d12-c787628d6d6a.png)

**Comment by enricosada** [comment]
@gavdraper can you try `dotnet build`  after `dotnet restore`?

**Comment by gavdraper** [comment]
Ah that errors, so I guess DotNet Run was hiding the exception. I'm now getting 

```
Gavins-MacBook-Air:fsharpnew gavindraper$ dotnet build
Compiling fsharpnew for DNXCore,Version=v5.0

Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/gavindraper/fsharpnew/"/Users/gavindraper/fsharpnew/obj/Debug/dnxcore50"/dotnet-compile.assemblyinfo.fs'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Interop.CheckIo[TSafeHandle](TSafeHandle handle, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
   at System.IO.UnixFileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)
   at System.IO.UnixFileSystem.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, FileStream parent)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding)
   at Microsoft.DotNet.Tools.Compiler.Fsc.CompileFscCommand.Run(String[] args)
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args)
   at Microsoft.DotNet.Cli.Program.Main(String[] args)
/usr/local/share/dotnet/bin/dotnet "compile-fsc" "@/Users/gavindraper/fsharpnew/obj/Debug/dnxcore50/dotnet-compile.rsp" returned Exit Code 134
/Users/gavindraper/fsharpnew/project.json(14,32): warning NU1007: Dependency specified was NETStandard.Library >= 1.0.0-rc2-23728 but ended up with NETStandard.Library 1.0.0-rc2-23804.

Compilation failed.
    1 Warning(s)
    0 Error(s)

Time elapsed 00:00:00.9050829

Gavins-MacBook-Air:fsharpnew gavindraper$

```

**Comment by enricosada** [comment]
ok, that's better. i'll try with mac.
thx for issue, let me check :smile:

**Comment by enricosada** [comment]
btw @gavdraper, i think `dotnet run` hide errors if `dotnet-compile` throw an exception is another bug (not related to f#) /cc @piotrpMSFT @blackdwarf
I'll check why dotnet-compile-fsc throw the exception.

**Comment by davidoptima** [comment]
I tried the same steps on windows and get an error, could be unrelated

![image](https://cloud.githubusercontent.com/assets/10268540/12890887/7cef0790-ce7d-11e5-9267-58b6438563f6.png)

**Comment by enricosada** [comment]
windows error it's about quoting of argument, waiting merge of pr #1285 to fix it.
maybe in osx the bug is another, i'll try in travis tonight

**Comment by muratg** [comment]
I hit the same issue on OS X. Last Thursday's build.

**Comment by piotrpMSFT** [comment]
Fixed in #1285
